cloudapp/src/app/requested-resources-service/requested-resources-service.ts

```diff
@@ -0,0 +1,250 @@
+
```

Comment:
Please rename requested-resources-services to requested-resources. This will give us a nice place to put other services as part of #16.